apps/angular/projection/src/app/ui/list-item/list-item.component.ts

```diff
@@ -18,18 +15,7 @@ import { CardType } from '../../model/card.model';
 export class ListItemComponent {
   @Input() id!: number;
   @Input() name!: string;
-  @Input() type!: CardType;
+  @Output() onDelete = new EventEmitter<number>();
```

Comment:
id and name are useless. Use ng-content to delegate the display to the parent

Reply:
Didn't think about the inputs being useless, you're right. Updated it.